res/xml/preferences.xml

```diff
@@ -50,6 +50,18 @@
         </PreferenceScreen>
     </PreferenceCategory>
 
+    <PreferenceCategory
+	    android:title="@string/p_units_title">
+            
+        <de.duenndns.ListPreferenceWithValue
+		android:key="p.units"
```

Comment:
The key should be `p_units` for consistency with the other prefs in the main prefs screen.